src/main/java/com/TableTalk/Enterprise/dao/RoomServiceDAOStub.java

```diff
@@ -1,41 +1,49 @@
-/*package com.TableTalk.Enterprise.dao;
+package com.TableTalk.Enterprise.dao;
 
 import com.TableTalk.Enterprise.dto.Room;
-import com.TableTalk.Enterprise.dto.User;
 
-import java.util.*;
-import java.util.stream.Stream;
+import java.util.HashMap;
+import java.util.List;
 
 public class RoomServiceDAOStub implements IRoomServiceDAO {
 
     HashMap<String, Room> allRooms = new HashMap<>();
 
 
     @Override
-    public Room fetch(String id) {
-
+    public void fetch(String id) {
+        //TODO: Add your function code
     }
 
     @Override
     public void delete(String id) throws Exception {
-
+        //TODO: Add your function code
     }
 
     @Override
     public Room save(Room room) throws Exception {
         String roomID = room.getId();
-        allRooms.put(roomID,room);
+        allRooms.put(roomID, room);
         return room;
     }
 
     @Override
     public List<Room> fetchAvailableRooms(String userID) {
-        List<Room> availableRooms = new ArrayList<Room>();
-        Set<String> roomIDSet = allRooms.keySet();
-        Stream<String> roomStream = roomIDSet.stream();
-        roomStream.
+        return null;
+    }
 
+//    @Override
+//    public List<Room> fetchAvailableRooms(String userID) {
+//        List<Room> availableRooms = new ArrayList<Room>();
+//        Set<String> roomIDSet = allRooms.keySet();
+//        Stream<String> roomStream = roomIDSet.stream();
+//        roomStream.
+//
+//    }
```

Comment:
Committing commented out code is considered bad practice in version control, because version control offers better alternatives:
1) Use a feature branch for experimental features.
2) Delete what is not needed.  You can always compare with history to see what used to be there.
Commented out code is considered clutter in a version control system.